packages/mintbase-wallet/package.json

```diff
@@ -22,6 +22,6 @@
   },
   "homepage": "https://github.com/near/wallet-selector/tree/main/packages/mintbase-wallet",
   "peerDependencies": {
-    "near-api-js": "^1.0.0 || ^2.0.0"
+    "near-api-js": "3.0.3"
```

Comment:
The `peerDependencies` version for NAJ should be consistent across all packages in wallet-selector.

I am not aware of what changes were made in v3.x but it looks like they were breaking changes.

I am worried this could either cause strange issues in dApps or the package managers would complain when installing NAJ because this version will conflict with other version ranges...

Reply:
`One thing I noticed when signing transactions that require confirmation on the wallet I am not being redirected back to the dApp.
`

this is already fixed

Reply:
Appreciate the review, will try downgrade the version and report back

Reply:
@kujtimprenkuSQA  already downgraded

Reply:
Let's revert this change then on the peerDependencies for near-api-js.

Reply:
Done